src/encoded/tests/test_audit_file.py

```diff
@@ -912,3 +912,41 @@ def test_audit_self_matching_md5sum(testapp, file7):
         errors_list.extend(errors[error_type])
     assert any(error['category'] == 'inconsistent matching_md5sum'
                for error in errors_list)
+
+
+def test_audit_incorrect_index(testapp, fastq_index, single_fastq_indexed):
```

Comment:
I think you should also include a case where index_of is associated correctly and make sure the audit is not triggered.

Reply:
I've added a test for this, thank you